packages/redux/src/recentlyViewed/types/action.types.ts

```diff
@@ -12,7 +12,7 @@ interface FetchRecentlyViewedProductsSuccessAction extends Action {
 }
 interface FetchRecentlyViewedProductsFailureAction extends Action {
   type: typeof actionTypes.FETCH_RECENTLY_VIEWED_PRODUCTS_FAILURE;
-  payload: { error: Error };
+  payload: { error: Error | unknown };
```

Comment:
`Error | unknown` is effectively `unknown` as you can see here: https://github.com/microsoft/TypeScript/blob/b975dfa1027d1f3073fa7cbe6f7045bf4c882785/tests/baselines/reference/unknownType1.types

If you want to remove a warning, then it should be typed as `unknown`. However, that might need other code changes related with this issue https://github.com/Farfetch/blackout/issues/114